INFO:root:Program started
INFO:root:Robot set up!
INFO:root:Odometry was started, PID: 8637:
INFO:root:Initial location: 0.2, 1.8, 3.141592653589793
WARNING:root:Traceback (most recent call last):
  File "/home/pi/oscar/main.py", line 56, in main
    oscar.navigateMap([0, 4], [4, 1])
  File "/home/pi/oscar/Robot.py", line 129, in navigateMap
    if not ((grid(possible_cell + watchout_1) == -1) and (grid(possible_cell + watchout_2) == -1)): # SI NO ESTAN OCUPADAS LAS DOS
TypeError: 'numpy.ndarray' object is not callable

WARNING:root:Abruptly stopped the robot!
INFO:root:Odometry was stopped
